src/ProjectNeon/Assets/Features/Battle/Cards/EnemyCardSelection.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyCardSelection : MonoBehaviour
+{
+    [SerializeField] private CardResolutionZone resolutionZone;
+    [SerializeField] private GameEvent onEnemyTurnsConfirmed;
+    [SerializeField] private BattleState battle;
+
+
+    private List<Enemy> enemies = new List<Enemy>();
+    /**
+     * @todo #190:30min Refactor the relation Member - Enemy. At the moment we can't extract
+     *  enemies from BattleState object because they are all wrapped in Member instances.
+     */
+
```

Comment:
@SilasReinagel I don't know why github removed your previous comment about the field initialization, but here's what I've faced: we use a list of `Enemy` in `EnemyCardSelection`, but we doesn't have this list; instead we have a `List<Member>` in `BattleState.GetEnemies()`. So I've left a puzzle so we can wire the correct class into this and get the expected enemies from `BattleState`.

It seems that we have a lot of double use of classes in Party, Character and Enemy relations; IMHO we should focus all this is just two constructs, `Member` and `List<Member>` and what would differ them from enemy characters, AI-allied characters and player-controlled characters would be the `TurnAI` class (which actually is the play algorithm for some Member). So we would rename `TurnAI` to something more adequate (like `PLayerAlgorithm` or something like that) and then create some `HumanPlayer` classes that implements `Play() : PlayedCard`, and in this implementation its await for the player to play its cards, WDYT?

Reply:
In general I agree. 

I have a feeling that Player Characters and Enemies will still be fundamentally different, even if they adhere to a `Member` interface of some sort. 

It's too early to tell.